src/renderer/reader/components/picker/SearchFormPicker.tsx

```diff
@@ -118,6 +120,19 @@ class SearchFormPicker extends React.Component<IProps, IState> {
                             <SVG ariaHidden={true} svg={searchIcon} />
                     }
                 </button>
+                {this.state.inputValue ?
+                <button
+                    className={stylesGlobal.button_clear}
+                    type="reset"
+                    onClick={() => {
+                        this.inputRef.current.value = "";
```

Comment:
Code guard: `this.inputRef.current?.value =`

Reply:
TypeScript does not allow this: "La partie gauche d'une expression d'assignation ne doit pas être un accès à une propriété facultative.ts(2779)"